tests/test-libraries/libraries/Makefile

```
@@ -35,6 +35,7 @@ $(1)_$(3)_TARGETS += \
 	.libs/$(3)/lib$(1).a \
 	.libs/$(3)/lib$(1).o \
 	.libs/$(3)/lib$(1).dylib \
+	.libs/$(3)/lib$(1).so
```

This makes the diff smaller the next time a file is add to this list:

```suggestion
	.libs/$(3)/lib$(1).so \
```